src/ucp/api/ucp.h

```diff
@@ -211,14 +211,23 @@ enum ucp_ep_params_field {
  * @ref ucp_ep_create() function.
  */
 enum ucp_ep_params_flags_field {
-    UCP_EP_PARAMS_FLAGS_CLIENT_SERVER  = UCS_BIT(0)   /**< Using a client-server
+    UCP_EP_PARAMS_FLAGS_CLIENT_SERVER  = UCS_BIT(0),  /**< Using a client-server
                                                            connection establishment
                                                            mechanism.
                                                            @ref ucs_sock_addr_t
                                                            sockaddr field
                                                            must be provided and
                                                            contain the address
                                                            of the remote peer */
+    UCP_EP_PARAMS_FLAGS_NO_LOOPBACK    = UCS_BIT(1)   /**< Avoid connecting the
+                                                           endpoint to itself,
+                                                           when connecting the
+                                                           endpoint to the same
+                                                           worker it's created on.
```

Comment:
Avoid connecting the endpoint to itself when connecting the endpoint to the same worker it was created on.